src/database.js

```diff
@@ -119,8 +133,8 @@ class Database {
   }
 }
 
-const configureKnex = async (config) => {
-  return Knex(config)
+const configureKnex = async (config, buffer = []) => {
+  return instrumentKnex(Knex(config), { logger: console, buffer: buffer, enabled: config.instrument })
```

Comment:
Should this not be configurable, i.e. we should have some config when creating the DB class to enable/disable the instrumentation?

Reply:
@mdebarros the `instrumentKnex` function takes an `enabled` flag. If it's true then it will instrument otherwise it will just return the given knex instance. There's an extra `instrument` variables in the db config that is used to toggle the instrumenting.